hazelcast-enterprise/maven.functions.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# THIS FILE IS DUPLICATED AND MUST BE KEPT IN SYNC MANUALLY
```

Comment:
1. May be use symlink on the fly?
2. Or may be this https://stackoverflow.com/a/34392052. so cd into the script directry and call docker build passing the Dockerfile filename. Apparently paths are relative to the current directory

Reply:
> May be use symlink on the fly?

I tried that, Dockers too smart and won't take it

> Or may be this https://stackoverflow.com/a/34392052. so cd into the script directry and call docker build passing the Dockerfile filename. Apparently paths are relative to the current directory

I wanted to keep the image buildable with nothing more than a simple `docker build` command, without any special arguments etc, so that I wouldn't have to worry about existing callers (e.g. the vulnerability scanner).

> ouch. painful :(

This was the "least worst" option, IMO :(

Reply:
Got it. Lets leave it and not rock the boat so to speak :)

Also ,just realized that devs do build image locally I am sure so this will confuse them or worst, break things